mail/mail.mjs

```diff
@@ -0,0 +1,185 @@
+export default function template (bookName, action) {
+return `<!DOCTYPE html>
+
+<html>
+<head>
+<meta content="text/html; charset=utf-8" http-equiv="Content-Type"/>
+<meta content="width=device-width" name="viewport"/>
+<meta content="IE=edge" http-equiv="X-UA-Compatible"/>
+<link href="https://fonts.googleapis.com/css?family=Montserrat" rel="stylesheet" type="text/css"/>
+<link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet" type="text/css"/>
+<style type="text/css">
```

Comment:
It would be cool to have the css file apart

Reply:
Yeah it would be really cool haha, but I do not know what the path to link the css file would be, anyway I will check it